### ISSUE TITLE
Add ExtendedPrivKey  and ExtendPubKey classes 

### ISSUE THREAD

**Issue by CerberusVT** [bitcoindevkit/bdk-ffi#138]
Hi!

The Kotlin library only contains `ExtendedKeyInfo`.
It would be very useful to also have `ExtendedPrivKey` and `ExtendPubKey`.

**Comment by notmandatory** [comment]
Hi, can you explain a bit more about how you want to use `ExtendedPrivKey` and `ExtendPubKey` if we expose them in the Kotlin library? It would help with prioritizing and maybe there's another way to do what you need.

**Comment by notmandatory** [comment]
OK yes I understand now and this is a very good suggestion. But to make your example work we don't need new `ExtendedPrivKey` and `ExtendPubKey` classes.  Would it work for you if we add an `xpub` property to the `ExtendedKeyInfo` class? 

The FFI definition would look like this:
```udl
dictionary ExtendedKeyInfo {
  string mnemonic;
  string xprv;
  string xpub;
  string fingerprint;
};
```

**Comment by notmandatory** [comment]
I'm open to other solutions if you have a need for something more. I just realized an existing PR #139 also adds xpub to `ExtendedKeyInfo`.  If no objections I'm going to work on getting that one merged and then we also solve this issue.

**Comment by notmandatory** [comment]
This should be fixed as part of #154 which is focused on deriving xpubs from xprv keys.

**Comment by notmandatory** [comment]
@elmarsan please checkout new API in #154, it was just merged and will be in the next release if you want to play with it in your project.
